PLGMAG2V2-287: Officially support OneStepCheckout.com (#1633)
MultiSafepay/docs: content/payments/integrations/ecommerce-platforms/magento2/faq/supported-magento2-checkouts.md

read_more: "."
---

MultiSafepay supports multiple checkouts:

{{< details title="Magento 2 core checkout" >}}

> Our plugin works out of the box with the Magento 2 core checkout which is based on the Luma theme.

Reviewer: ```suggestion
Our plugin works out of the box with the Magento 2 core checkout, which is based on the Luma theme.
```

{{< /details >}}

{{< details title="OneStepCheckout.com" >}}
MultiSafepay is an official partner of [OneStepCheckout.com](https://www.onestepcheckout.com/).

> Our plugin works with the latest version of the OneStepCheckout extension. If you have questions or notice missing features or bugs when both plugins are installed, feel free to contact our integration team at [integration@multisafepay.com](mailto:integration@multisafepay.com)

Reviewer: ```suggestion
Our plugin works with the latest version of the OneStepCheckout extension. If you have questions or notice missing features or bugs when both plugins are installed, email the Integration Team at [integration@multisafepay.com](mailto:integration@multisafepay.com)
```

{{< /details >}}

{{< details title="Other checkouts" >}}

> Our plugin has been tested with the following checkouts:

Reviewer: ```suggestion
We have tested our plugin with the following checkouts:
```
